packages/map-template/src/components/Search/Search.scss

```diff
@@ -49,6 +51,25 @@
         }
     }
 
+    &__back {
+        &--button {
```

Comment:
`&-button` since this is not a BEM modifier.

That probably also explains this 😁 

![image](https://github.com/user-attachments/assets/ac725db7-0eb0-4c90-9a36-3492fdf431ea)